frontend/src/store/FamLoginUserState.ts

```diff
@@ -207,7 +207,61 @@ const getCachedAppRolesForDelegatedAdmin = (
             ? -1
             : 1;
     });
-}
+};
+
+const getMyCachedPermissions = () => {
```

Comment:
and let's rename this to getMyAdminPermission, cause later on we'll need to get regular permission as well, thanks!